File: C/3rd Week/swap.c
// 14. Swap two numbers without using third variable
#include <stdio.h>

void main()
{
    int a = 2, b = 3;
    printf("Before swap:\na = %i\nb = %i\n", a, b);
    a = a + b;
    b = a - b;
    a = a - b;
    printf("After swap:\na = %i\nb = %i\n", a, b);
}

File: java Programming/1st Week/swap.java
// 5. Write a Program to swap two numbers using bitwise operators

import java.util.Scanner;

public class swap {
   public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      System.out.print("Enter two numbers: ");
      int a = sc.nextInt();
      int b = sc.nextInt();
      sc.close();
      System.out.println("Before swapping: a = " + a + ", b = " + b);
      a = a ^ b;
      b = a ^ b;
      a = a ^ b;
      System.out.println("After swapping : a = " + a + ", b = " + b);
   }
}
